client/components/SignIn.jsx

```diff
@@ -22,7 +22,13 @@ class SignIn extends React.Component {
       <div className='sign-in'>
         <section className='section'>
           <div className='content'>
-            <h1>Sign in</h1>
+            <h1>Student Sign-in</h1>
+            <button className='button is-primary'><a href='http://localhost:3000/auth/github'>Continue with GitHub</a></button>
```

Comment:
Let's put the `<a>` on its own line.
We also need to use an environment variable in `.env` for the host so we only use `http://localhost:3000` while we're in a development (not production).